.github/workflows/main.yml

```diff
@@ -76,16 +75,16 @@ jobs:
           server: https://deploy.octopus.app
           service_account_id: 6e5497b3-8427-42b4-a354-e05ea095f7c1
 
-      - name: Push to Octopus 🐙
-        if: (! contains(github.ref, '/merge')) && (! contains(github.ref, '/dependabot/')) && (! contains(github.ref, 'prettybot/'))
+      - name: Push Packages to Octopus Deploy 🐙
```

Comment:
Minor renaming and reordering so it diffs better with GHA workflows from our other repos